config/pmd/ruleset.xml

```diff
@@ -41,6 +43,8 @@
     <exclude name="UseUtilityClass"/>
     <exclude name="LoosePackageCoupling"/>
     <exclude name="DataClass"/>
+    <exclude name="CognitiveComplexity"/>
+    <exclude name="CyclomaticComplexity"/>
```

Comment:
Do you have an example of code that got flagged by this?
Again in general would be good if we don't turn these off centrally.

Reply:
I removed it